azure/blobs_logs_monitoring/README.md

```diff
@@ -1,46 +1,4 @@
-# Datadog-Azure function
+# Note: Deprecated in favor of the new blob storage based forwarder
```

Comment:
Could we keep documentation and put a big warning at the top?
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts